contracts/transmuter/Storage.sol

```diff
@@ -64,70 +65,70 @@ enum WhitelistType {
+    address facetAddress; // Facet contract address
+    bytes4[] functionSelectors; // Ex. bytes4(keccak256("transfer(address,uint256)"))
 }
 
 struct FacetInfo {
-    address facetAddress;                        // Facet contract address
-    uint16 selectorPosition;                     // Position in the list of all selectors
+    address facetAddress; // Facet contract address
+    uint16 selectorPosition; // Position in the list of all selectors
 }
 
 struct DiamondStorage {
-    bytes4[] selectors;                          // List of all available selectors
-    mapping(bytes4 => FacetInfo) selectorInfo;   // Selector to (address, position in list)
-    IAccessControlManager accessControlManager;  // Contract handling access control
+    bytes4[] selectors; // List of all available selectors
+    mapping(bytes4 => FacetInfo) selectorInfo; // Selector to (address, position in list)
+    IAccessControlManager accessControlManager; // Contract handling access control
 }
 
 struct ImplementationStorage {
-    address implementation;                      // Dummy implementation address for Etherscan usability
+    address implementation; // Dummy implementation address for Etherscan usability
 }
 
 struct ManagerStorage {
-    IERC20[] subCollaterals;                     // Subtokens handled by the manager or strategies
-    bytes config;                                // Additional configuration data
+    IERC20[] subCollaterals; // Subtokens handled by the manager or strategies
+    bytes config; // Additional configuration data
 }
 
 struct Collateral {
-    uint8 isManaged;                             // If the collateral is managed through external strategies
-    uint8 isMintLive;                            // If minting from this asset is unpaused
-    uint8 isBurnLive;                            // If burning to this asset is unpaused
-    uint8 decimals;                              // IERC20Metadata(collateral).decimals()
-    uint8 onlyWhitelisted;                       // If only whitelisted addresses can burn or redeem for this token
-    uint216 normalizedStables;                   // Normalized amount of stablecoins issued from this collateral
-    uint64[] xFeeMint;                           // Increasing exposures in [0,BASE_9[
-    int64[] yFeeMint;                            // Mint fees at the exposures specified in `xFeeMint`
-    uint64[] xFeeBurn;                           // Decreasing exposures in ]0,BASE_9]
-    int64[] yFeeBurn;                            // Burn fees at the exposures specified in `xFeeBurn`
-    bytes oracleConfig;                          // Data about the oracle used for the collateral
-    bytes whitelistData;                         // For whitelisted collateral, data used to verify whitelists
-    ManagerStorage managerData;                  // For managed collateral, data used to handle the strategies
+    uint8 isManaged; // If the collateral is managed through external strategies
```

Comment:
why not keeping the indentation?

Reply:
I just saved the file and it changed it to this, I guess that's the prettier version. I can revert